Document/0x05g-Testing-Network-Communication.md

```diff
@@ -15,6 +50,76 @@ Make sure that the hostname and the certificate itself are verified correctly. E
 
 ### Static Analysis
 
+#### Trust Anchors in the Network Security Configuration
+
+When running on Android 7.0 (API level 24) or higher, apps targeting those API levels will use a default Network Security Configuration that doesn't trust any user supplied CAs, reducing the possibility of MITM attacks by luring users to install malicious CAs. However, this protection can be bypassed by using a custom Network Security Configuration with a custom trust anchor indicating that the app will trust user supplied CAs.
+
+Use a decompiler (e.g. jadx or apktool) to confirm the target SDK version. After decoding the app you can look for the presence of `targetSDK` present in the file apktool.yml that was created in the output folder.
+
+The Network Security Configuration should be analyzed to determine what settings are configured. The file is located inside the APK in the /res/xml/ folder with the name network_security_config.xml.
```

Comment:
That's actually not necessary. You can call it whatever you want. So what about
```suggestion
The Network Security Configuration should be analyzed to determine what settings are configured. The name of the file can be found in the `android:networkSecurityConfig` attribute of the `<application>` tag inside the Android Manifest. The file can then be found in the /res/xml/ folder with the .xml extension.
```
Though it's possible to store it in other folders as well, if you really want to I think.